studio-docs/source/github-integration.md

```diff
@@ -54,30 +55,36 @@ jobs:
             apollo service:push
           fi
 ```
+
 ### Github Actions
-```
+
+```yaml:title=.github/workflows/schema_check.yaml
 name: Check
 
 # Controls when the action will run. Triggers schema validation on every pull request against main
+
 on:
-  pull_request:
-    branches: [ main ]
-    
+pull_request:
+branches: [ main ]
+
 jobs:
-  schema-check:
-    # The type of runner that the job will run on any runner with node will work
-    runs-on: ubuntu-latest
+schema-check: # The type of runner that the job will run on any runner with node will work
+runs-on: ubuntu-latest
 
     # Steps represent a sequence of tasks that will be executed as part of the job
     steps:
     - uses: actions/checkout@v2
+      with:
+        # Use the head of the PR instead of the merge commit
+        ref: ${{ github.event.pull_request.head.sha }}
```

Comment:
I think we do want to use the merge commit for schema reasons right? Specifically if I'm behind master I don't want to run a check on an old schema and get a spurious diff right?

Reply:
@jsegaran Are you suggesting monorepo is doing it wrong? :)